dominio/src/test/dominioTest/MapaTest.java

```diff
@@ -78,5 +73,52 @@ public void bomberNoSePuedeMoverAUnObstaculo() {
 		Assert.assertEquals(bombers.get(0).getPosicionX(), 1, 0.000001);
 	}
 	
-
+	@Test
+	public void bomberEstaEnUnaEsquina() {
+		mapita.a�adirBomber(new Bomber(1, 1));
+		ArrayList<Bomber> bombers;
+		bombers = mapita.getBombers();
+		Assert.assertEquals(1, bombers.size());
+		Assert.assertEquals(bombers.get(0).getPosicionX(), 1, 0.000001);
+		mapita.moverBomber(bombers.get(0), -1,0);
+		Assert.assertEquals(bombers.get(0).getPosicionX(), 1, 0.000001);
+	}
+	
+	@Test
+	public void a�adirBomba() {
+		mapita.a�adirBomba(new Bomba(3,3));
+		Entidad[][] entidades = mapita.getEntidades();
+		Assert.assertTrue(entidades[3][3] instanceof Bomba);
+	}
+	
+	@Test
+	public void explotarBomba() throws InterruptedException {
+		mapita.a�adirBomba(new Bomba(3,3));
+		Entidad[][] entidades = mapita.getEntidades();
+		Assert.assertTrue(entidades[3][3] instanceof Bomba);
+		Bomba bombita = (Bomba) entidades[3][3];
+		Thread.sleep(bombita.getTiempoExplosion()*1000);
+		bombita.explotar();
+		Assert.assertTrue(bombita.isExploto());
+	}
+	
+	@Test
+	public void explotarBombaAlLadoDeBomber() {
+		mapita.a�adirBomba(new Bomba(3,3));
+		mapita.a�adirBomber(new Bomber(4,4));		
+		ArrayList<Bomber> bombers;
+		bombers = mapita.getBombers();
+		Entidad[][] entidades = mapita.getEntidades();
+		Assert.assertTrue(entidades[3][3] instanceof Bomba);
+		Bomba bombita = (Bomba) entidades[3][3];
+		bombita.explotar();
+		for(Bomber bomber: bombers) {
+			if((Math.abs(bomber.getPosicionX()-bombita.getPosicionX()) <= bombita.getRango()) && (Math.abs(bomber.getPosicionY()-bombita.getPosicionY()) <= bombita.getRango())) {
+				bomber.morir();
```

Comment:
se debería encapsular esta lógica

Reply:
Si, me sonaba raro mientras lo hacía. Pero, como Videla había dicho que la lógica la manejaramos desde los test, lo dejé así. Desde Mapa podría ponerse esta lógica?

Reply:
Claro, creo el juego más adelante va a ir verificando si hay bombas que explotaron, cuando esto pase se van a sacar de la matriz y podría ejecutarse el metodo "explotarBomba" en la clase mapa, que destruya lo que tenga que destruir, pasándole una bomba.

Reply:
Entonces: 

`bombita.explotar();
for(Bomber bomber: bombers) {
			if((Math.abs(bomber.getPosicionX()-bombita.getPosicionX()) <= bombita.getRango()) && (Math.abs(bomber.getPosicionY()-bombita.getPosicionY()) <= bombita.getRango())) {
				bomber.morir();
			}
		}`

Podría pasar a ser

`bombita.explotar();
mapita.explotarBomba(bombita);
`

o algo similar.

Reply:
Dale, estoy de acuerdo